error-handler.js

```diff
@@ -98,15 +99,16 @@ var mixIn = require('mout/object/mixIn'),
     stream.pipe(res);
   },
 
-  send = function send(statusCode, err, res, o) {
+  send = function send(statusCode, err, res, o) {    
+    if (!err.message) err.message = statusCodes[statusCode];
```

Comment:
Oh, mout!

In this version, error name is included by default, i.e. even if no serializer is defined. This is new behavior. Also, any non-enumerables or non-own properties on err are not given to serializer, except for the explicit exception of err.name. Seems kind of like a one-off.

Let me throw it back to you for a second...why is it improper to mutate the err object at this point? We are not calling next(err)... Is anything else going to inspect the err object after we handle this error? Moreover, we are adding a few properties only if they are falsy ('' for err.message and undefined for anything else), not overwriting anything. Are we mutating the error object, or possibly just augmenting it as we handle it?